client/state/action-types.js

```diff
@@ -587,10 +587,6 @@ export const MEDIA_SOURCE_CHANGE = 'MEDIA_SOURCE_CHANGE';
 export const MEDIA_SET_NEXT_PAGE_HANDLE = 'MEDIA_SET_NEXT_PAGE_HANDLE';
 export const MEDIA_SET_QUERY = 'MEDIA_SET_QUERY';
 export const MEDIA_CLEAR_SITE = 'MEDIA_CLEAR_SITE';
-export const MEMBERSHIPS_CONNECTED_ACCOUNTS_LIST = 'MEMBERSHIPS_CONNECTED_ACCOUNTS_LIST';
-export const MEMBERSHIPS_CONNECTED_ACCOUNTS_STRIPE_AUTHORIZE_REQUEST =
-	'MEMBERSHIPS_CONNECTED_ACCOUNTS_STRIPE_AUTHORIZE_REQUEST';
-export const MEMBERSHIPS_CONNECTED_ACCOUNTS_RECEIVE = 'MEMBERSHIPS_CONNECTED_ACCOUNTS_RECEIVE';
 export const MEMBERSHIPS_CONNECTED_ACCOUNTS_STRIPE_DISCONNECT_SUCCESS =
 	'MEMBERSHIPS_CONNECTED_ACCOUNTS_STRIPE_DISCONNECT_SUCCESS';
```

Comment:
This action could also be easily removed. It's identical to what `requestSettings` (`MEMBERSHIPS_SETTINGS`) does: refetch the membership settings for the site after Stripe was disconnected.

Reply:
Nice find 👍  I'll follow-up in another PR to remove that one.

Reply:
See #47619 for removing the unnecessary action.